git/git_intro.qmd

```diff
@@ -23,6 +23,29 @@ git clone https://github.com/user/repo
 
 ## File staging and git commit
 
+Staging in Git involves adding new, modified, or deleted files to a staging area before committing them. This allows for flexibility in choosing the files to commit.
+
+1. Check status via `git status` You'll see what branch you are on (which for new repos will be master) and status of files (untracked, modified, or deleted).  
```

Comment:
new repos get "main" instead of "master"

Reply:
I got it from here: 
https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell#:~:text=A%20branch%20in%20Git%20is,branch%20pointer%20moves%20forward%20automatically.
and here:
https://www.reddit.com/r/git/comments/14gcbx3/can_i_change_the_default_branch_from_master_to/

But I can also remove that part, as its probably freely configurable.

Reply:
Oh interesting, it must be a version thing then. On my mac (git v 2.39.3) when I init a repo it gets "main", but on NAC cluster (git v2.17.1) it gets "master". It doesn't really matter though as you said this is configurable.